README.md

```diff
@@ -1,8 +1,50 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Adaobi Aniuchi . Jarvis Consulting
 
+I am a graduate student at the University of Windsor in the final days of my program. I am pursuing a professional master's degree in Applied Computing. Before that, I interned first as a Microsoft Leap Apprentice and subsequently as a graduate intern at software development and consulting firm. During this time, I collaborated with the software team to release features for a custody management system for a local bank. The diverse range of applications in different industries possible by applying software development principles is one of my favourite things about the industry. I also like that it is an ever-evolving field, so there is always something new to learn. I am currently with Jarvis to expand and gain new software and data engineering skills by working on various software projects using different languages and frameworks in an agile environment.
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Shell, HTML/CSS, Javascript/Node.js/React.js, RDBMS/SQL, Git, Agile/Scrum
+
+**Competent:** Spring Boot, C#, ASP.Net Core, MongoDB, Mocha, Chai, Sinon
+
+**Familiar:** Neo4J, NUnit, Mockito, Django, Azure DevOps
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AdaobiAniuchi](https://github.com/jarviscanada/jarvis_data_eng_AdaobiAniuchi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AdaobiAniuchi/tree/master/linux_sql)]: Developed a Minimum Viable Product MVP of a Linux Cluster Monitoring Agent for managing and monitoring hardware information and resource usage for nodes in a Linux cluster. Implemented the project using Bash scripting, Docker, PostgreSQL and Crontab. Tested the app on the command-line and verified the results manually from the database. Managed all bash and SQL script source code with Git and GitHub. Provisioned the PostgreSQL database using Docker. Scheduled the bash agent program to run at one-minute intervals using Crontab.
```

Comment:
Developed a ~Minimum Viable Product~ MVP of a Linux Cluster Monitoring Agent for managing and monitoring hardware information and resource usage for nodes in a Linux cluster. Implemented the project using Bash scripting, Docker, PostgreSQL and Crontab. Tested the app on the command-line and verified the results manually from the database. Managed all bash and SQL script source code with Git and GitHub. Provisioned the PostgreSQL database using Docker. Scheduled the bash agent program to run at one-minute intervals using Crontab.